extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I guess `package-lock` file should be also commited

Reply:
we don't do that in ps extensions at all